### PR TITLE
Add default IDE settings to project

### PR DESCRIPTION
**Pull request by ASDFGamer**
This adds the following default settings to the project:
- Default Code Style
- Disable some unused Warnings

It adds also suggested settings to the readme.md that could be used so that all contributors have similar settings.
These settings can't be synchronized, because they are saved in files that contain private information

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,59 @@
...
+### Tools
+
+Actions on Save:
+
+- Reformat code: On; All file types, Changed lines
+- Optimize imports: On; All file types
+
+Terminal:
+
+- Audible bell: On
```
Comment (ku-we) [inline README.md:52]: Should the bell be on? ;)

**Review by ku-we — APPROVED** [review]
LGTM. Let's try out these IDE settings. If needed, we can adjust or fine tune them in the future. You should take a look at my two comments, I think two manual IDE settings are not yet correctly documented.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,59 @@
...
+- Rearrange code: On
+- Optimize imports: On
+- Analyze code: On
+
+### Editor
+
+Design Tools:
+
+- Default Editor Mode: Drawables: Split
+- Default Editor Mode: Drawables: Design
```
[the remark this answers is not in the document]
Comment (ASDFGamer) [reply README.md]: Yes, that was a copy paste error. It is fixed with commit d62a745

File: README.md (hunk 1 above)
Comment (ASDFGamer) [reply README.md]: No, it is fixed with commit https://github.com/d3kad3nt/deCONZ-SunriseClock/commit/d62a745429172c41a6810c6572d2c4edc46be3b7

**Pull request merged**